src/browser/Page.zig

```diff
@@ -1025,17 +1025,12 @@ pub fn getElementByIdFromNode(self: *Page, node: *Node, id: []const u8) ?*Elemen
     return null;
 }
 
-pub fn registerMutationObserver(self: *Page, observer: *MutationObserver) !void {
-    try self._mutation_observers.append(self.arena, observer);
+pub inline fn registerMutationObserver(self: *Page, observer: *MutationObserver) !void {
```

Comment:
I think it's worth avoid `inline` unless we can prove it's worth it.

https://ziggit.dev/t/inlining-functions/1341/3